src/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx

```diff
@@ -298,9 +332,22 @@ const getProps = (props: Props) => {
   }
 }
 
-const renderPricing = (salePrice, saleDate, user, mediator, size) => {
+const renderPricing = (
+  salePrice,
+  saleDate,
+  user,
+  mediator,
+  size,
+  filtersHaveUpdated,
+  paginated
+) => {
   const textSize = size === "xs" ? "2" : "3t"
-  if (user) {
+
+  // If user is logged in we show prices. Otherwise we show prices only for the default view - on page 1 and filters not changed.
```

Comment:
Thanks for adding a note on the functionality intention here. There is a lot of nuance around this behavior